integration-tests/webjars-locator/pom.xml

```diff
@@ -25,9 +25,8 @@
             <artifactId>quarkus-webjars-locator</artifactId>
         </dependency>
         <dependency>
-          <groupId>org.webjars</groupId>
-          <artifactId>jquery</artifactId>
-          <version>3.4.1</version>
+            <groupId>org.webjars</groupId>
+            <artifactId>jquery</artifactId>
```

Comment:
I think I would keep the hardcoded version here. Otherwise we will have to upgrade the test for each upgrade with no real value.

Reply:
Ok, I'll revert